.cbmc-batch/jobs/aws_byte_buf_write_from_whole_string/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags:        "--bounds-check;--div-by-zero-check;--float-overflow-check;--nan-check;--pointer-check;--pointer-overflow-check;--signed-overflow-check;--undefined-shift-check;--unsigned-overflow-check;--unwind;1;--unwinding-assertions;--unwindset;memcmp.0:17,strlen.0:17,memcpy_impl.0:17,memset_impl.0:17"
```

Comment:
Unnecessary space at the beginning.

Reply:
These are autogenerated by the makefile.  Figuring out how to modify the makefile to avoid the extra space will be a backlog item.